### PR TITLE
Update app and runtime

### LINKED ISSUE #6: Doesn't build, didn't find mono5 in 19.08
https://flathub.org/builds/#/builders/35/builds/2003

```
FB: Running: flatpak --user info runtime/org.gnome.Sdk/x86_64/3.36 --show-metadata
Dependency Extension: org.freedesktop.Sdk.Extension.mono5 19.08
FB: Running: flatpak --user info org.freedesktop.Sdk.Extension.mono5/x86_64/19.08 --show-commit
Installing org.freedesktop.Sdk.Extension.mono5/x86_64/19.08 from flathub
FB: Running: flatpak --user install flathub -y --noninteractive org.freedesktop.Sdk.Extension.mono5/x86_64/19.08
error: Nothing matches org.freedesktop.Sdk.Extension.mono5 in remote flathub
Error installing deps: running `flatpak --user install flathub -y --noninteractive org.freedesktop.Sdk.Extension.mono5/x86_64/19.08`: Child process exited with code 1
```

This needs https://github.com/flathub/org.freedesktop.Sdk.Extension.mono5/issues/7 to be fixed.

### LINKED ISSUE #7: Update runtime to 3.38

### PR DESCRIPTION
**Pull request by wjt**
Fixes #6 inasmuch as it builds.
Fixes #7 except a newer runtime.

### CODE REVIEW

File: org.gnome.gbrainy.json [hunk 1]
```diff
@@ -3,56 +3,68 @@
     "sdk": "org.gnome.Sdk",
     "branch": "stable",
     "runtime": "org.gnome.Platform",
-    "runtime-version": "3.36",
+    "runtime-version": "41",
```
Comment (hfiguiere) [inline org.gnome.gbrainy.json]: `42` is the latest.

File: org.gnome.gbrainy.json [hunk 2]
```diff
@@ -3,56 +3,68 @@
...
+    "cleanup": [
+        "/lib/monodoc",
+        "/lib/pkgconfig"
+    ],
     "modules": [
         {
             "name": "gtk-sharp3",
-            "config-opts": [
-                           ],
+            "config-opts": [],
```
Comment (hfiguiere) [inline org.gnome.gbrainy.json]: you could as well remove it.

```suggestion
```

File: org.gnome.gbrainy.json [hunk 3]
```diff
@@ -3,56 +3,68 @@
...
     ],
-    "build-options" : {
-        "append-path": "/usr/lib/sdk/mono5/bin",
-        "append-ld-library-path": "/usr/lib/sdk/mono5/lib",
+    "build-options": {
+        "append-path": "/usr/lib/sdk/mono6/bin",
+        "append-ld-library-path": "/usr/lib/sdk/mono6/lib",
         "env": {
-            "PKG_CONFIG_PATH": "/app/lib/pkgconfig:/app/share/pkgconfig:/usr/lib/pkgconfig:/usr/share/pkgconfig:/usr/lib/sdk/mono5/lib/pkgconfig"
+            "PKG_CONFIG_PATH": "/app/lib/pkgconfig:/app/share/pkgconfig:/usr/lib/pkgconfig:/usr/share/pkgconfig:/usr/lib/sdk/mono6/lib/pkgconfig"
```
Comment (hfiguiere) [inline org.gnome.gbrainy.json]: `"append-pkg-config-path": "/usr/lib/sdk/mono6/lib/pkgconfig"` is probably better here.

**Review by hfiguiere — CHANGES_REQUESTED** [review]

File: org.gnome.gbrainy.json (hunk 3 above)
Comment (hfiguiere) [reply org.gnome.gbrainy.json]: just to clarify it would be one level up out of the `env` object.

File: org.gnome.gbrainy.json (hunk 1 above)
Comment (wjt) [reply org.gnome.gbrainy.json]: Good catcH!

**Comment by wjt** [comment]
Cracked it!

**Comment by wjt** [comment]
Test build works on my machine™

**Comment by jordimas** [comment]
Thanks, really appreciated

**Pull request merged**